.codecov.yml

```diff
@@ -0,0 +1,28 @@
+# "documentation": https://gist.github.com/stevepeak/53bee7b2c326b24a9b4a
+coverage:
+  range: 50..100
+  round: down
+  precision: 0
+
+  status:
+    project: false
+    patch: false
+    changes: false
+
+# https://docs.codecov.io/docs/pull-request-comments
+comment:
+  layout: "diff"           # "diff, flags, files"
+  behavior: default
+  require_changes: false   # if true: only post the comment if coverage changes
+  require_base: true       # [yes :: must have a base report to post]
+  require_head: true       # [yes :: must have a head report to post]
+  branches: []             # branch names that can post comment
+
+ignore:
+  - "cbor_gen.go"
```

Comment:
Please ignore the support and gen directories too (alternatively, only include actors)